### PR TITLE
Replace category properties with methods

### PR DESCRIPTION
**Pull request by medidatakerry**
This addresses issue #13

### CODE REVIEW

**Comment by medidatakerry** [comment]
This PR's merge is dependent on PR #17

File: README.md [hunk 1]
```diff
@@ -8,11 +8,15 @@ Contact us at mork@mdsol.com
 
 ## Properties
 
-MORK adds a property to the ResearchKit `ORKTaskResult` class via a Category:
+MORK adds a method to the ResearchKit `ORKCollectionResult` class via a
+category:
 
-#### `ORKTaskResult.mork_fieldDataFromResults`
+#### `-(NSArray *)mork_fieldDataFromResults;`
```
Comment (jtomson-mdsol) [inline README.md]: this should be `mork_getFieldDataFromResults` as well?

Comment (medidatakerry) [inline README.md]: Ah, good eye there, batter. :baseball:

**Comment by medidatakerry** [comment]
Updated.

**Comment by jtomson-mdsol** [comment]
LGTM

**Pull request merged**